core/templates/dev/head/pages/error/error.html

```diff
@@ -4,15 +4,7 @@
 {% endblock navbar_breadcrumb %}
 
 {% block maintitle %}
-  {% if status_code == 400 %}
-    I18N_ERROR_PAGE_TITLE_400
-  {% elif status_code == 401 %}
-    I18N_ERROR_PAGE_TITLE_401
-  {% elif status_code == 404 %}
-    I18N_ERROR_PAGE_TITLE_404
-  {% elif status_code == 500 %}
-    I18N_ERROR_PAGE_TITLE_500
-  {% endif %}
+  {{ error_page_title }}
```

Comment:
We still have jinja template!

Reply:
Do you mean to refer to just replacing this line by `<[ error_page_title ]>` or do you refer to the `{% block maintitle %}`. If it's the latter, I don't think it is possible to do it easily without a major rewrite across all files(if you have an idea without changing much of the rest of templates, please suggest it ). I thought the original task(in #3950) meant replacing only the `if` condition.

Also, fixed the test failure(not sure how I missed amending, my apologies).

Reply:
Through this PR we are expecting to remove the jinja template!
Currently, I don't have any suggestion how you can do this, but I think the "Base controller" can help you regarding this (not sure!).

Reply:
Replacing `{% block maintitle %}` is a rather non-trivial task since:
1. If angular is indeed used to do anything, since title is bound to `rootScope`, it is going to be a bit messy.
2. A simple method may be to just change it would be just use js `document.title` but according to [this](https://angular.io/guide/set-document-title), "That's dirty and undermines your chances of running the app outside of a browser someday." so I don't know if that is something desirable either.

Regardless of the approach, this seems like a much larger change(and would need to be changed in *all* existing templates I believe) I would prefer to discuss before implementing anything(given by unfamiliarity with angular, so sorry if I'm wrong).

Reply:
Let's leave maintitle for now. I agree it's a bigger change and is best tackled as a separate issue altogether.

Reply:
@DubeySandeep : ping, are there any other changes necessary then?